sale_product_pack/models/sale_order_line.py

```diff
@@ -73,11 +73,20 @@ def expand_pack_line(self, write=False):
             if vals_list:
                 self.create(vals_list)
 
-    @api.model
-    def create(self, vals):
-        record = super().create(vals)
-        record.expand_pack_line()
-        return record
+    @api.model_create_multi
+    def create(self, vals_list):
+        new_vals = []
+        res = self.browse()
+        for elem in vals_list:
+            product = self.env["product.product"].browse(elem["product_id"])
+            if product.pack_ok and product.pack_type == "detailed":
```

Comment:
@augusto-weiss Hi, i read, your module and mine. I think that's a bad idea to add a method who returns all pack_type for the create because we can't use it in the other condition. You should overwrite the create, split the records in self into two categories, the first one for the super() call and the second one, for the work you want for your product pack. What do you think about this?

Reply:
https://github.com/acsone/product-pack/pull/2